configs/webpack/base.coffee

```diff
@@ -1,16 +1,66 @@
 path = require 'path'
 
+_ = require 'lodash'
 webpack = require 'webpack'
 ExtractTextPlugin = require 'extract-text-webpack-plugin'
-webpackUMDExternal = require 'webpack-umd-external'
 
-_ = require 'lodash'
+LOADERS =
+  babel: 'babel'
+  json: 'json'
+  file: 'file?name=[name].[ext]'
+  url: 'url?limit=30000&name=[name]-[hash].[ext]'
+  coffee: 'coffee'
+  cjsx: 'coffee-jsx'
+  lessCompiled: ExtractTextPlugin.extract('css!less')
+  style: 'style'
+  css: 'css'
+  less: 'less'
+
+RESOLVABLES =
+  js: { test: /\.js$/, exclude: /node_modules/, loader: LOADERS.babel}
+  json: { test: /\.json$/,   loader: LOADERS.json }
+  jsx: { test: /\.jsx$/, loader: LOADERS.babel}
+  coffee: { test: /\.coffee$/,  loader: LOADERS.coffee }
+  cjsx: { test: /\.cjsx$/,    loader: LOADERS.cjsx }
+
+STATICS =
+  json: RESOLVABLES.json
+  image: { test: /\.(png|jpg|svg|gif)/, loader: LOADERS.file }
+  font: { test: /\.(woff|woff2|eot|ttf)/, loader: LOADERS.url }
+
+BASE_BUILD =
+  # TODO
+  #   Loading .js through babel is not yet working.
+  #   It doesn't seem to exclude node_modules,
```

Comment:
I've added an `exclude: [/node_modules/]` entry to the loader in https://github.com/philschatz/gh-board/blob/master/webpack.config.js#L27 and that seems to work